tools/bin/sdc-sbupload

```diff
@@ -75,30 +80,30 @@ load_sdc_sysinfo
 SBAPI_CREDS="${CONFIG_sbapi_http_user}:${CONFIG_sbapi_http_pass}"
 SBAPI_URL="${CONFIG_sbapi_url}"
 
-JOYENT_KEY=${DIR}/../share/servicebundle/pubkey.key
+SUPPORT_KEY=${DIR}/../share/servicebundle/pubkey.key
 
-DC_INFO="$(echo "${CONFIG_datacenter_company_name}" | tr -cd [:alnum:] | tr [:upper:] [:lower:])"
-if [[ "$CONFIG_coal" == "true" ]]; then
+DC_INFO="$(echo "${CONFIG_datacenter_company_name}" | tr -cd "[:alnum:]" | tr "[:upper:]" "[:lower:]")"
+if [[ "${CONFIG_coal:-false}" == "true" ]]; then
     # If this is COAL we want it to be identified as such, regardless of the
     # DC name they choose for their coal.
     DC_INFO=${DC_INFO}.coal
 else
-    DC_INFO=${DC_INFO}.${CONFIG_datacenter_name}
+    DC_INFO=${DC_INFO}.${CONFIG_datacenter_name:?}
 fi
-ENCRYPTED_FILE="$(dirname ${FILENAME})/${BUNDLE_TYPE}${DC_INFO}.${SYSINFO_UUID}.$(TZ=UTC date "+%Y%m%dT%H%M%SZ")"
+ENCRYPTED_FILE="$(dirname "${FILENAME}")/${ISSUE_ID}${BUNDLE_TYPE}${DC_INFO}.${SYSINFO_UUID}.$(TZ=UTC date "+%Y%m%dT%H%M%SZ")"
 
 #
 # Encrypt the support bundle
```

Comment:
One last nit... Maybe a comment line below here?

```
# We use uuencode to preserve the filename because we have no idea what
# the file name is when we decrypt it.
```